metrics/web/test/auth/presentation/pages/login_page_test.dart

```diff
@@ -129,6 +152,9 @@ void main() {
 
 /// A testbed widget, used to test the [LoginPage] widget.
 class _LoginPageTestbed extends StatelessWidget {
+  /// A [GlobalKey] needed to get the current context of the [LoginPage].
+  static final GlobalKey childKey = GlobalKey();
```

Comment:
```suggestion
  /// A [GlobalKey] for the [LoginPage] to use in tests.
  static final GlobalKey loginKey = GlobalKey();
```

And I think it would be better to pass the key as a parameter in the constructor.